api/api/specs/routers.yaml

```diff
@@ -923,6 +932,8 @@ components:
         - ensembler_id
         - resource_request
         - timeout
+        - env
+        - secrets
```

Comment:
Strangely `env` wasn't specified as a required field for pyfunc ensemblers, but it was for docker ensemblers. I'm adding it here just to make things consistent.